language-sql.md

```diff
@@ -3,11 +3,33 @@
 ### Minimum Requirements
 
 - MySQL keywords in uppercase
-- Column names in lowercase
+    E.g.: 
+    ```
+    UPDATE `consumer` SET `PASSWORD` = MD5(`PASSWORD`);
+    ```
+
+- Database table names must be plural. 
+    E.g.: 
+    `users`, `subscriptions`
+    
+    for Linking tables :
+    `users_subscriptions`
```

Comment:
I've already given my thumb for this, by personally I would prefer singular names -- it's what I've been used to. Much less confusing, and rolls easier on the tongue.

For example, a table that holds properties is called `property`, a table that holds users is called `user`, and a table that might map a property to a user (regardless of the mapping involved, M2M or OTM) is also a singular underscore singular `user_property`. 

Though whatever you guys decide to go with would be fine for me. :D